client/my-sites/site-settings/analytics/form-google-analytics.jsx

```diff
@@ -88,8 +90,9 @@ export const GoogleAnalyticsForm = ( props ) => {
 		placeholderText,
 		recordSupportLinkClick,
 		setDisplayForm,
+		isSEOEligible,
 	};
-	if ( props.siteIsJetpack ) {
+	if ( props.siteIsJetpack && isSEOEligible ) {
```

Comment:
Same here, I don't understand why the SEO eligibility is affecting the GA eligibility.

Reply:
This is a check for which Analytics form to show. For Atomic, we show the Jetpack form. But for free Atomic, we want to show the Simple form with WordPress.com plan upgrade nudges. We could change it to an Analytics eligibility check.

Reply:
> We could change it to an Analytics eligibility check.

👍

Reply:
Changed here: https://github.com/Automattic/wp-calypso/pull/53196/commits/1cf6ee4b7abc92aeaa02d91ca239f8992bf10e1a